extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ amp-story-dev-tools-tab-preview {
+  --i-amphtml-story-dev-tools-device-background: #101010 !important;
+  --i-amphtml-story-dev-tools-device-screen: #2E3137 !important;
+  --i-amphtml-story-dev-tools-device-shadow: 0px 11.0046px 27.5114px rgba(0, 0, 0, 0.34), inset 0px 2.20091px 3.30137px rgba(44, 46, 52, 0.6) !important;
+  --i-amphtml-story-dev-tools-device-chip-background: #2e2e2e !important;
+  --i-amphtml-story-dev-tools-device-chip-background-hover: #3d3d3d !important;
+  --i-amphtml-story-dev-tools-device-chip-icon-background: black !important;
+
+  --i-amphtml-story-dev-tools-device-dialog-background: #111111cc !important;
+  --i-amphtml-story-dev-tools-device-dialog-help-url-border: #ffffff22;
+  --i-amphtml-story-dev-tools-device-dialog-help-url-color: #ffffff77;
```

Comment:
The main advantage to using CSS variables is that they're reusable. By using name like `device-screen` you can't reuse this variable for anything else.
Could you please make these work so we don't have to add a new variable anytime we add a new element?